local/flake.nix

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Why does this subflake exist?

Reply:
Please remove it to reduce this project complexity

Reply:
Changed in 69d33b662eb5ef0df1326c2a013d9d062e7e79e1 to align with `std` (for now).

Reply:
flake.nix still exists, the only difference is that I have no idea what it does now